### PR TITLE
Make bracket-matcher work on the same line

### LINKED ISSUE #2: Bracket matching is not highlighted properly
Bracket matching is not highlighted properly with atom 18 +,  python3 and dracula theme
Image of dracula

But it is highlighted properly with atom dark fore example:
Image of atom dark

### PR DESCRIPTION
**Pull request by ghost**
Fixes #2.

### CODE REVIEW

File: styles/base.less [hunk 1]
```diff
@@ -302,4 +302,9 @@ atom-text-editor::shadow .highlight.current-result .region,
 atom-text-editor::shadow .highlight.current-result ~ .highlight.selection .region {
   background: lighten(@dark-gray, 20%); // to distinguish from the other results
   z-index: -1000;
-}
\ No newline at end of file
+}
+
+atom-text-editor::shadow  .bracket-matcher .region {
```
Comment (nesl247) [inline styles/base.less]: You've got an extra space between the first and second element.

Comment (ghost) [inline styles/base.less]: Had no effect but I fixed it, thanks.

Comment (nesl247) [inline styles/base.less]: I know, I'm just anal about these things. I review PRs all day, so it carries over to other projects :)

**Comment by JimmyMultani** [comment]
Amazing, thank you!

**Pull request merged**